floodviz/static/js/hydrograph.js

```diff
@@ -143,39 +141,43 @@
 				update(data);
 			});
 		};
-    
+
 		/**
-		 * Handle all mouse over movements for calling element. 
+		 * Displays tooltip for hydrograph at a datapoint in addition to
+		 * corresponding map site tooltip
 		 */
-		self.mouseover = function(d) {
+		self.series_tooltip_show = function(d) {
 			d3.select(d.data.name).classed("gage--hover", true);
 			focus.attr("transform", "translate(" + x(d.data.time_mili) + "," + y(d.data.value) + ")");
 			focus.select("text").html(d.data.key + ": " + d.data.value + " cfs " + " " + d.data.time + " " + d.data.timezone);
-			// Interative linking with map
-			FV.map_figure.mousemove(d.data.name, d.data.key);
+			// Interactive linking with map
+			FV.map_figure.site_tooltip_show(d.data.name, d.data.key);
```

Comment:
No, FV.map_figure.site_tooltip_show would not be calling FV.hydrograph_figure.series_tooltip_show. James calls a separate function in FV.hydrograph to handle this line accent, so no infinite loop. What do you think about calling the FV.map_figure functions in the events themselves like so:

```
       .on("mouseover", function(d) {
	    FV.map_figure.site_tooltip_show(d.data.name, d.data.key);
	    return self.series_tooltip_show(d);
	})
```